src/v2/Components/DownloadAppBadge.tsx

```diff
@@ -18,6 +23,14 @@ const DOWNLOAD_IOS_APP_BADGE =
 const DOWNLOAD_ANDROID_APP_BADGE =
   "https://files.artsy.net/images/download-android-app.svg"
 
+interface AnalyticsOptions {
+  context_module: ContextModule
+  context_page_owner_id?: string
+  context_page_owner_slug?: string
+  destination_path: string
+  context_page_owner_type: PageOwnerType
+}
+
```

Comment:
Let's avoid defining this type here and in theDownloadAppBadge file - we should be able to get what we need using the ClickedAppDownload interface exported by Cohesion.

Instead, we could use one of the following types:
`const options: Partial<ClickedAppDownload> = { ... }` 
This basically means "the type of `options` must be a subset of `ClickedAppDownload`." So `options` could have some of, or all of, the fields specified in `ClickedAppDownload`, but we couldn't add _new_ fields to it.

`const options: Omit<ClickedAppDownload, "action" | "subject"> = { ... }`
This means "we expect `options` to have all of the properties of `ClickedAppDownload` except for `action` and `submit`

Reply:
Or see my other comment for another suggestion on typing 🤷‍♂️

Reply:
Applied the changes suggested on https://github.com/artsy/force/pull/8859/files?diff=unified&w=0#r751185851